R/oauth-init.R

```diff
@@ -85,13 +87,16 @@ init_oauth2.0 <- function(endpoint, app, scope = NULL, type = NULL,
   }
 
   # Use authorisation code to get (temporary) access token
-  req <- POST(endpoint$access, encode = "form",
-    body = list(
-      client_id = app$key,
-      client_secret = app$secret,
-      redirect_uri = redirect_uri,
-      grant_type = "authorization_code",
-      code = code))
+  req_params <- c(
```

Comment:
I think it would be slightly better to use `stats::modifyList()` and change the order of the argument so that `user_params` "loses" if a name is defined in both places.